tools/release.sh

```diff
@@ -7,7 +7,7 @@ Usage() {
     exit 1
 }
 
-if [ "`dirname $0`" != "tools" ] ; then
+if [ "`dirname $0`" != "./tools" ] ; then
```

Comment:
If you don't have `.` in your `PATH` and you're trying to run an executable in your current dictory you need the leading `./`, however, if you're providing a path to an executable in another directory (as is the case here), I believe it works fine.
```
bmw@localhost:~/Development/certbot/certbot$ pwd
/home/bmw/Development/certbot/certbot
bmw@localhost:~/Development/certbot/certbot$ echo $PATH
/home/bmw/bin:/home/bmw/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
bmw@localhost:~/Development/certbot/certbot$ ls
acme            certbot-compatibility-test  CONTRIBUTING.md     docs                letsencrypt-auto-source         LICENSE.txt        readthedocs.org.requirements.txt  tools
build           certbot.log                 docker-compose.yml  examples            letsencrypt-compatibility-test  linter_plugin.py   releases                          tox.cover.sh
certbot         certbot-nginx               Dockerfile          ISSUE_TEMPLATE.md   letsencrypt-nginx               linter_plugin.pyc  setup.cfg                         tox.ini
certbot-apache  CHANGELOG.md                Dockerfile-dev      letsencrypt-apache  letshelp-certbot                MANIFEST.in        setup.py                          venv
certbot-auto    CHANGES.rst                 Dockerfile-old      letsencrypt-auto    letshelp-letsencrypt            README.rst         tests                             venv3
bmw@localhost:~/Development/certbot/certbot$ certbot-auto
certbot-auto: command not found
bmw@localhost:~/Development/certbot/certbot$ tools/release.sh  
++ dirname tools/release.sh
...
```